test/onmt/ExtendedCmdLineTest.lua

```diff
@@ -100,8 +100,11 @@ end
 function cmdLineTest.fail_unknown()
   local cmd = onmt.utils.ExtendedCmdLine.new()
   onmt.Seq2Seq.declareOpts(cmd)
-
+  onmt.data.SampledDataset.declareOpts(cmd)
+  -- will generate error
+  print('')
```

Comment:
This is no more needed as of https://github.com/OpenNMT/OpenNMT/commit/e794053462d6473b35aa8d5d75bbfaf16ce31c79.